README.md

```diff
@@ -350,18 +350,71 @@ We expect to add Monero into the ports tree in the near future, which will aid i
 
 ### On OpenBSD:
 
-This has been tested on OpenBSD 5.8.
+This has been tested on OpenBSD 6.2.
```

Comment:
Did you test it to be non working on 5.8 ?

Reply:
No, but 5.8 is not supported by OpenBSD anymore. I thought I'll just update the Version to the latest -stable, where I tested the build.

Reply:
What does supported mean in this context ?

Reply:
Well I could just leave the OpenBSD 5.8 section and add a new one for 6.2.

Reply:
I'd rather that, unless you have evidence the 5.8 instructions would not work.